style.css

```diff
@@ -0,0 +1,97 @@
+/* stylelint-disable length-zero-no-unit */
+.container {
+  height: 590px;
+  width: 375px;
+  background: #1c1a19;
+  background-image: url("./svg/illustration-header-mobile.svg");
+  background-repeat: no-repeat;
+  background-position: right 0 bottom -50px;
+  background-size: 35%;
+}
+
+h1 {
+  width: 324px;
+  height: 124px;
```

Comment:
- [ ] [OPTIONAL] Kindly try to refrain from using fixed widths on containers as much as possible as this can cause problems in responsivity and cause bugs in the design.

Reply:
@ehs9nino, No problem. Happy to help :)